### PR TITLE
Better comments in activity sidebar

### PR DESCRIPTION
**Pull request by LukasReschke**
- [x] https://github.com/nextcloud/activity/pull/49

Downstreaming of https://github.com/owncloud/core/pull/24275

cc @nickvergessen as you wrote most of it

### CODE REVIEW

File: apps/comments/js/activitytabviewplugin.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
+ * @author Joas Schilling <nickvergessen@owncloud.com>
```
Comment (nickvergessen) [inline apps/comments/js/activitytabviewplugin.js]: :/

**Comment by nickvergessen** [comment]
Also requires a patch in the activity app, will have a look at that.

**Comment by nickvergessen** [comment]
Requires https://github.com/nextcloud/activity/pull/49 to have a visual impact (multi-line and long comments are trimmed to 3 lines and can be expended with a click)

**Comment by nickvergessen** [comment]
👍, second review @rullzer ?

**Comment by MorrisJobke** [comment]
Tested and works :

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**